tests/test_FBbasis2D.py

```diff
@@ -17,133 +16,131 @@
 DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
 
 
```

Comment:
I'd prefer to see a small comment here just saying these represent `(resolution_pixels, dtype)` or something to that effect.

I understand this is found in the code line(s) below, but if that gets moved around or edited it will be easier on the next developer to state the variables.

Reply:
Added a comment describing the parameter tuples